common/lib/driver-definitions/src/storage.ts

```diff
@@ -98,6 +98,11 @@ export interface IDocumentStorageServicePolicies {
     // If this policy is provided, it tells runtime on ideal size for blobs
     // Blobs that are smaller than that size should be aggregated into bigger blobs
     readonly minBlobSize?: number;
+
+    /**
+     * This policy tells the runtime that the driver will not use cached snapshots older than this value.
+     */
+    readonly snapshotCacheExpiryTimeoutMs?: number;
```

Comment:
Sorry for delay, but it feels like the name is not that appropriate.
From consumer POV (loader, runtime), term "timeout" does not seem appropriate.
Language of "maximum cache duration" or something similar seems more on point describing what this value represents.

And while it's more complicated topic, I feel like we still do not have an answer to - how all of that would work for someone like Whiteboard, who does not want GC (maybe for now, maybe forever) - can and should runtime in such cases be allowed to use cache of any duration?

Reply:
Tyler updated the name here - https://github.com/microsoft/FluidFramework/pull/8806#event-5920886956

For the second part, I created an issue - https://github.com/microsoft/FluidFramework/issues/8818. Please take a look and you can more context / detail if needed.